include/proteus/Frontend/Func.hpp

```diff
@@ -237,6 +238,19 @@ class FuncBase {
     Function *F = getFunction();
     F->setName(Name);
   }
+
+  // Cast the given Var's value to type T and return a new Var holding
+  // the converted value.
+  template <typename T>
+  std::enable_if_t<std::is_arithmetic_v<T>, Var &> cast(Var &V) {
```

Comment:
`cast` != convert. Maybe we want a more descriptive name for this operation. Why not just convert?

Reply:
I have changed it to `convert`. I wonder if `convertTo` is even better? It delivers high clarity:
```c++
 auto &I2 = F.convertTo<int>(FfromI);
```